src/renderer/components/layout/tab-layout.tsx

```diff
@@ -23,17 +21,13 @@ interface Props {
 
 export const TabLayout = observer(({ className, contentClass, tabs, children }: Props) => {
   const routePath = navigation.location.pathname;
-  const cluster = getHostedCluster();
-  if (!cluster) {
-    return null; // fix: skip render when removing active (visible) cluster
```

Comment:
Why this is removed? It was part of #928 which fixed something 🙂

Reply:
I have tried to reproduce what is said in the comment, works fine.
It's removed primarily to make the component "pure" and as a result it can be exposed via `@lens/extension` as common component.
Regarding to #928 it doesn't affect scrollbar at all.

Reply:
Well, it's not fully pure at the moment since it use `const routePath = navigation.location.pathname` from external `import`.. :) but it can be fixed easily with react `context`  or moving it up to `props`.